local-setup/docs/TENANT-ONBOARDING.md

```
@@ -0,0 +1,352 @@
+  `egov-enc-service` has no key for a tenant that wasn't registered yet. After the
+  tenant exists: `docker restart egov-enc-service`, then re-run with only
+  `employee_file`.
+- **Employee jurisdiction** lands on the wrong root (the loader scopes to the
+  first boundary root) → fix directly:
+  ```sql
+  UPDATE eg_hrms_jurisdiction j SET boundary='<intended-root-code>'
+  FROM eg_hrms_employee e WHERE e.uuid = j.employeeid AND e.tenantid = '<tenant>';
+  ```
+- **MapConfig not written.** Unlike the configurator wizard's Phase 2, the
+  headless path does **not** write `RAINMAKER-PGR.MapConfig`, so the citizen map
+  has no boundary source. Seed it (see the [map post-config](#after-onboarding-enable-the-map--boundaries) below).
```

markdownlint-cli2 (0.23.0)

[warning] Link fragments should be valid

(MD051, link-fragments)